fe/fe-core/src/main/java/org/apache/doris/planner/SingleNodePlanner.java

```diff
@@ -1159,6 +1159,10 @@ private PlanNode createAggregationPlan(SelectStmt selectStmt, Analyzer analyzer,
                     aggInfo.getSecondPhaseDistinctAggInfo());
             newRoot.init(analyzer);
             Preconditions.checkState(newRoot.hasValidStats());
+        } else if (aggInfo.isMetaScan()) {
+            Preconditions.checkState(root instanceof MetaScanNode);
+            MetaScanNode metaScanNode = (MetaScanNode) root;
+            metaScanNode.pullDictSlots(aggInfo);
```

Comment:
we need to check agg function name and then do `pullDictSlots`

Reply:
I think we not actually need check here currently,  have done this in the analyze phase